test/test_intensify/test_successive_halving.py

```diff
@@ -149,33 +147,109 @@ def test_top_k_2(self):
             test _top_k() for configs with different instance-seed-budget keys
         """
         intensifier = SuccessiveHalving(
-            tae_runner=None, stats=self.stats,
-            traj_logger=TrajLogger(output_dir=None, stats=self.stats),
+            tae_runner=None, stats=self.stats, traj_logger=None,
             rng=np.random.RandomState(12345),
             instances=[1, 2], initial_budget=1)
         self.rh.add(config=self.config1, cost=1, time=1,
-                    status=StatusType.SUCCESS, instance_id=1,
-                    seed=None,
+                    status=StatusType.SUCCESS, instance_id=1, seed=None,
                     additional_info=None)
         self.rh.add(config=self.config2, cost=10, time=10,
-                    status=StatusType.SUCCESS, instance_id=2,
-                    seed=None,
+                    status=StatusType.SUCCESS, instance_id=2, seed=None,
                     additional_info=None)
 
         with self.assertRaisesRegex(AssertionError, 'Cannot compare configs'):
             intensifier._top_k(configs=[self.config2, self.config1, self.config3],
                                k=1, run_history=self.rh)
 
+    def test_get_next_challenger_2(self):
+        """
+            test get_next_challenger for a presently running configuration
+        """
+        intensifier = SuccessiveHalving(
+            tae_runner=None, stats=self.stats, traj_logger=None,
```

Comment:
Is the `tae_runner` actually still a necessary argument here (probably also true for the `traj_logger`)?

Reply:
Yes. We just don't pass any during test cases when we know that will not be used.